control/lane_departure_checker/include/lane_departure_checker/lane_departure_checker.hpp

```diff
@@ -78,13 +78,19 @@ struct Input
   std::vector<std::string> boundary_types_to_detect{};
 };
 
+struct DerivativeDeviation
+{
+  double longitudinal_vel{0.0};
+};
```

Comment:
For one variable, I think there is no need to create a struct.

Reply:
fixed in 821cf29de7f95895292c453da6570cf537339121